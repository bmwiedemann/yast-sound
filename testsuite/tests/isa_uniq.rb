# encoding: utf-8

module Yast
  class IsaUniqClient < Client
    def main
      Yast.import "UI"
      # testedfiles: Sound.ycp sound/routines.ycp
      Yast.include self, "testsuite.rb"
      @READ_I = { "target" => { "size" => -1, "tmpdir" => "/tmp" } }

      TESTSUITE_INIT([@READ_I], nil)
      Yast.include self, "sound/routines.rb"

      TEST(lambda { isa_uniq }, [], nil)

      nil
    end
  end
end

Yast::IsaUniqClient.new.main
