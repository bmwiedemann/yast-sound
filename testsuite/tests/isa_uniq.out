Return	uniq.unknown_key
